docker/stack.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+# Script useful for manage stack saps services life-cycle.
+# See usage for commands & options.
+
+set -o errexit;
+
+readonly ARCHIVER_REPO=wesleymonte/archiver
+readonly CATALOG_REPO=wesleymonte/catalog
+readonly DISPATCHER_REPO=wesleymonte/dispatcher
+readonly SCHEDULER_REPO=wesleymonte/scheduler
```

Comment:
In the future you guys should create a saps docker hub, so you could keep that containers there. @thiagomanel

Reply:
why not now?

Reply:
Done. [Dockerhub](https://hub.docker.com/orgs/ufcgsaps/repositories).